### ISSUE TITLE
Need to supress promise checks

### ISSUE THREAD

**Issue by skirankumars31** [playwright-community/eslint-plugin-playwright#430]
Im using 2.7.1 and its complaining about missing await for waitForRequest

const requestPromise = page.waitForRequest("https://example.com/resource");
await page.getByText("trigger request").click();
const request = await requestPromise;

**Comment by luukvhoudt** [comment]
Same goes for `2.7.0` seems like the rule `missing-playwright-await` got stricter. Here is another one which should be included in the test cases:

```ts
const results = await Promise.all([
	expect(locator).toBeVisible(),
	...(someCondition ? [page.waitForResponse('...')] : []),
]);
```
